R/modify-model-field.R

```diff
@@ -248,10 +269,35 @@ replace_bbi_args <- function(.mod, .bbi_args) {
   return(.mod)
 }
 
+#' @describeIn modify_model_field _Deprecaed_  Append new decisions to the one(s) in a model object and corresponding YAML
+#' @param .decisions Character vector to add to `decisions` field
+#' @export
+add_decisions <- function(.mod, .decisions) {
+  warning("The `decisions` field has been replaced by `notes`. At some point it will be removed. Please use `add_notes()` going forward.")
+  modify_model_field(
+    .mod = .mod,
+    .field = YAML_DECISIONS,
+    .value = .decisions,
+    .append = TRUE
+  )
+}
+
+#' @describeIn modify_model_field _Deprecaed_ Replaces `decisions` field in a model object and corresponding YAML with new values
+#' @param .decisions Character vector to use as replacement
+#' @export
+replace_decisions <- function(.mod, .decisions) {
```

Comment:
any reason we don't have remove_decision/replace_all_decisions functions here also?

Reply:
Because that function is already deprecated. The only reason `add_` and `replace_` are still there is so it doesn't break any existing user code that calls them. (Similar to discussion above, but I don't see any danger here, so when I did these I went with option 3 for those two specific functions)